x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_overview.tsx

```diff
@@ -43,6 +57,50 @@ export const AnalyticsCollectionOverview: React.FC<AnalyticsCollectionOverviewPr
         rightSideItems: [<AnalyticsCollectionToolbar />],
       }}
     >
+      {!hasEvents && (
```

Comment:
Maybe let's move this code to a separate component, because Overview component used mostly like a composition without specific logic

Reply:
Makes sense, thank you!